### PR TITLE
R4R: Update English doc for record module

### PR DESCRIPTION
**Pull request by yunkunrao**
Update English doc for record module for issue #571

### CODE REVIEW

File: docs/modules/record/query.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+### Query a record
+
+```shell
+iriscli record query --chain-id=test --record-id=MyRecordID
+```
+
+After that, you will get detail info for the record which has the specfied record ID.
+
+```txt
+{
```
Comment (HaoyangLiu) [inline docs/modules/record/query.md]:
```json is much better

File: docs/modules/record/README.md [hunk 2]
```diff
@@ -1,85 +1,32 @@
...
-1. On-Chain record for the text data
-2. On-Chain record for the text file (TODO)
-3. On-Chain governance for the record params (TODO)
+Record allows you to submit, query and download your records on chain.
 
-## Interaction Process
-
-### Record process
-
-1. Any users can initiate a record request. It will cost you some tokens. If there’s no record of the data on the existing chains, the request will be completed successfully and the relevant metadata will be recorded onchain. And you will be returned a record ID to confirm your ownership of the data.
```
Comment (shirleypyj) [inline docs/modules/record/README.md]: will be recorded on the chain

File: docs/modules/record/README.md [hunk 3]
```diff
@@ -1,85 +1,32 @@
...
-2. If any others initiate a record request for the same data, the request will be directly rejected and it will hint that the relevant record data has already existed.
-3. Any users can search/download onchain based on the record ID.
```
Comment (shirleypyj) [inline docs/modules/record/README.md]: on the chain

File: docs/modules/record/README.md [hunk 4]
```diff
@@ -1,85 +1,32 @@
...
-4. At present, the maximum amount of stored data at most 1K Bytes. In the future, the dynamic adjustment of parameters will be implemented in conjunction with the governance module.
```
Comment (shirleypyj) [inline docs/modules/record/README.md]: the maximum amount of stored data is 1K Bytes.

File: docs/modules/record/query.md (hunk 1 above)
Comment (yunkunrao) [reply docs/modules/record/query.md]: Got it.

File: docs/modules/record/README.md (hunk 2 above)
Comment (yunkunrao) [reply docs/modules/record/README.md]: Got it

File: docs/modules/record/README.md (hunk 3 above)
Comment (yunkunrao) [reply docs/modules/record/README.md]: Got it.

File: docs/modules/record/README.md (hunk 4 above)
Comment (yunkunrao) [reply docs/modules/record/README.md]: Got it.

**Pull request merged**